ts/session/sending/MessageWrapper.ts

```diff
@@ -4,7 +4,7 @@ import { MessageEncrypter } from '../crypto/MessageEncrypter';
 import { PubKey } from '../types';
 
 function encryptionBasedOnConversation(destination: PubKey) {
-  if (ConvoHub.use().get(destination.key)?.isClosedGroup()) {
+  if (PubKey.is03Pubkey(destination.key) || ConvoHub.use().get(destination.key)?.isClosedGroup()) {
```

Comment:
To confirm, does `isClosedGroup()` only support v2 group now? Otherwise you should you add  the 03 pubkey check inside of the function?

```tsx
  public isClosedGroup(): boolean {
    return Boolean(
      (this.get('type') === ConversationTypeEnum.GROUP && (PubKey.is05Pubkey(this.id) || PubKey.is03Pubkey(destination.key))) ||
        this.isClosedGroupV2() 
    );
```

Reply:
eventually we will get rid of this isClosedGroup. Once we get rid of legacy closed groups.
It will  be nice to not have to check from the DB that a pubkey is a for a group.
With the new groups, we can just check for the prefix. 
I don't really think that change is needed, it was already true. This is just me being too excited of eventually getting rid of legacy groups.